cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -46,6 +46,52 @@ type NodeGroupAutoscalingOptions struct {
 	ScaleDownUnreadyTime time.Duration
 }
 
+const (
```

Comment:
Since you're moving the consts here, they should be deleted from the file they used to live in.

Reply:
I agree we should have them in one place. I think they should live near ratio options. Deleted them from the other location.